rosdep/python.yaml

```diff
@@ -3890,11 +3894,13 @@ python3-ruamel.yaml:
     zesty: [python3-ruamel.yaml]
 python3-setuptools:
   debian: [python3-setuptools]
+  gentoo: [dev-python/setuptools]
   ubuntu: [python3-setuptools]
 python3-venv:
   debian: [python3-venv]
   ubuntu: [python3-venv]
 python3-yaml:
+  gentoo: [dev-python/pyyaml]
```

Comment:
Can you please add the missing debian entry here as well?

Reply:
Yep. Will do.